tests/mypy/modules/success.py

```diff
@@ -208,15 +208,29 @@ class PydanticTypes(BaseModel):
     # String
     my_strict_str: StrictStr = 'pika'
     # PyObject
-    # TODO: my_pyobject_str: PyObject = 'datetime.date'
+    my_pyobject_str: PyObject = 'datetime.date'  # type: ignore
     my_pyobject_callable: PyObject = date
     # UUID
     my_uuid1: UUID1 = UUID('a8098c1a-f86e-11da-bd1a-00112444be1e')
-    my_uuid1_str: UUID1 = 'a8098c1a-f86e-11da-bd1a-00112444be1e'
+    my_uuid1_str: UUID1 = 'a8098c1a-f86e-11da-bd1a-00112444be1e'  # type: ignore
     # Path
-    my_file_path: FilePath = Path('root') / 'myfile.txt'
-    my_file_path_str: FilePath = 'root/myfile.txt'
-    my_dir_path: DirectoryPath = Path('root') / 'mydir'
-    my_dir_path_str: DirectoryPath = 'root/mydir'
+    my_file_path: FilePath = Path(__file__)
+    my_file_path_str: FilePath = __file__  # type: ignore
+    my_dir_path: DirectoryPath = Path('.')
+    my_dir_path_str: DirectoryPath = '.'  # type: ignore
```

Comment:
I guess one day, when we solve https://github.com/koxudaxi/pydantic-pycharm-plugin/issues/36#issuecomment-618478815 we could update  the mypy plugin to avoid the need for all these `# type: ignore`s